### PR TITLE
Fix bnb chain bug

### PR DESCRIPTION
**Pull request by SidharthK2**
# BNB Chain does not appear in the linked wikis autocomplete

WIP

- BNB autocomplete appears when the prompt is deleted after typing completely.

image

image

## Linked issues

closes #00, closes #001, closes https://github.com/EveripediaNetwork/issues/issues/1846

### CODE REVIEW

**Review by Softdev1 — APPROVED** [review]

File: src/components/CreateWiki/EditorModals/EditWikiDetailsModal/LinkedWikisInput.tsx [hunk 1]
```diff
@@ -64,6 +64,9 @@ const LinkedWikisInput = ({ wiki }: { wiki: Wiki }) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [search])
 
+  console.log('search', search)
+  console.log('res: ', results)
```
Comment (kesar) [inline src/components/CreateWiki/EditorModals/EditWikiDetailsModal/LinkedWikisInput.tsx:68]: please dont merge console logs in prod

File: src/components/CreateWiki/EditorModals/EditWikiDetailsModal/LinkedWikisInput.tsx [hunk 2]
```diff
@@ -162,11 +168,9 @@ const LinkedWikisInput = ({ wiki }: { wiki: Wiki }) => {
                     m={0}
                     fontSize="xs"
                     key={`option-${result.id}`}
-                    value={result}
+                    value={result.title}
```
Comment (kesar) [inline src/components/CreateWiki/EditorModals/EditWikiDetailsModal/LinkedWikisInput.tsx:171]: i think here might not be the title what we need

Comment (Royal-lobster) [reply src/components/CreateWiki/EditorModals/EditWikiDetailsModal/LinkedWikisInput.tsx:171]: yes it should be id

Comment (Adebesin-Cell) [reply src/components/CreateWiki/EditorModals/EditWikiDetailsModal/LinkedWikisInput.tsx:171]: And the child is the value, weirdly the bug of not showing happens with ``BNB Chain``, prolly a thing with the id of the wiki which is weird.

**Pull request merged**